apps/learner/src/lib/components/LearningUnit.svelte

```diff
@@ -59,18 +59,16 @@
   </div>
 
   {#if showplayerpanel}
-    <div class="flex items-center justify-between">
+    <div class="flex items-center gap-x-3">
       <button
-        class="flex cursor-pointer items-center gap-x-2 rounded-full bg-slate-100 px-6 py-4 transition-colors hover:bg-slate-200"
+        class="flex cursor-pointer items-center gap-x-2 rounded-full border border-slate-950/10 px-6 py-4 transition-colors hover:bg-slate-200"
```

Comment:
Should we explicitly set the background color to white?

Since the background is white now, can consider using a lighter background on hover.

Reply:
Hmmm, just a random thought, most of our users will be using mobile phones. The hover effect does not work with mobile browsers if I'm not wrong.

Reply:
bg-slate 50 or 100 for the hover?
<img width="1728" height="1005" alt="Screenshot 2025-07-23 at 2 15 46 AM" src="https://github.com/user-attachments/assets/8f9b5c2e-6d5a-4ed1-ab3d-d49ad2422559" />

Reply:
> Hmmm, just a random thought, most of our users will be using mobile phones. The hover effect does not work with mobile browsers if I'm not wrong.

Yes, hover effect does not work with mobile browsers. But since we’re building a web app, it still makes sense to have them for desktop users.